test/e2e/antreaipam_test.go

```diff
@@ -48,7 +48,7 @@ var (
 				IPRanges: []crdv1alpha2.SubnetIPRange{{IPRange: crdv1alpha2.IPRange{
 					CIDR:  "",
 					Start: "192.168.240.100",
-					End:   "192.168.240.129",
+					End:   "192.168.240.106",
```

Comment:
Can we expand the "Dedicated IPPool" in e2e test? Otherwise it will affect the other test cases and will be hard to change.

Reply:
This range is sufficient for other tests as of today, but I can add another pool for sake of Expand test if you prefer. Was that your meaning?

Reply:
I mean you can use the pool-1 (Dedicated IPPool) to avoid the conflict with other cases, because currently only one case use this IPPool.

Reply:
I created a dedicated pool for the new test, to avoid possible conflicts in future tests.